docs/_redirects

```diff
@@ -0,0 +1,344 @@
+https://docs.directus.io/blog/getting-started-with-directus-and-preact.html                                                 https://directus.io/docs/tutorials
+https://docs.directus.io/blog/getting-started-with-directus-and-qwik-city.html                                              https://directus.io/docs/tutorials
+https://docs.directus.io/blog/getting-started-with-directus-and-remix.html                                                  https://directus.io/docs/tutorials
+https://docs.directus.io/blog/directus-seo-tips-tricks.html                                                                 https://directus.io/docs/tutorials/tips-and-tricks/search-engine-optimization-best-practices
+https://docs.directus.io/blog/getting-started-solidstart.html                                                               https://directus.io/docs/tutorials/getting-started/fetch-data-from-directus-with-solidstart
+https://docs.directus.io/blog/getting-started-directus-sveltekit.html                                                       https://directus.io/docs/tutorials/getting-started/fetch-data-from-directus-with-sveltekit
+https://docs.directus.io/blog/enrich-user-data-clearbit-directus-automate.html                                              https://directus.io/docs/tutorials/workflows/enrich-user-data-with-clearbit-and-directus-automate
+https://docs.directus.io/blog/devs-intro-to-composable.html                                                                 https://directus.io/docs/guides/extensions/app-extensions/composables
+https://docs.directus.io/blog/building-a-notebook-chrome-extension-with-directus.html                                       https://directus.io/docs/tutorials/projects/build-a-notebook-chrome-extension-with-directus-auth
+https://docs.directus.io/blog/building-a-support-system-in-the-directus-data-studio.html                                    https://directus.io/docs/tutorials
+https://docs.directus.io/blog/deploying-directus-to-azure-web-apps-with-docker.html                                         https://directus.io/docs/tutorials/self-hosting/deploy-directus-to-azure-web-apps
+https://docs.directus.io/blog/getting-started-with-directus-spring-boot.html                                                https://directus.io/docs/tutorials/getting-started/fetch-data-from-directus-with-spring-boot
+https://docs.directus.io/blog/google-docs-preview.html                                                                      https://directus.io/docs/tutorials/tips-and-tricks/preview-files-in-live-preview-with-google-docs-previews
+https://docs.directus.io/blog/automatic-transcripts-deepgram.html                                                           https://directus.io/docs/tutorials/workflows/generate-transcripts-with-deepgram-and-directus-automate
+https://docs.directus.io/blog/building-a-form-data-collection-and-email-notification-system-with-directus-and-next-js.html  
```

Comment:
Looks like we're missing the redirect link on this one. @hola-soy-milk did you want this link omitted or should we redirect to docs/tutorials?

```suggestion
```

OR

```suggestion
https://docs.directus.io/blog/building-a-form-data-collection-and-email-notification-system-with-directus-and-next-js.html  https://directus.io/docs/tutorials
```